_source/logzio_collections/_prometheus-sources/dotnet-helm-chart.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Ship .NET logs with Helm
+logo:
+  logofile: dotnet-logo.png
+  orientation: horizontal
+data-source: .NET over Helm
+data-for-product-source: Metrics
+templates: ["k8s-daemonset"]
+open-source:
+  - title: logzio-helm
+    github-repo: logzio-helm
+contributors:
+  - nshishkin
+shipping-tags:
+  - container
+order: 430
+---
+
+Helm is a tool for managing packages of pre-configured Kubernetes resources using Charts. This integration allows you to collect and ship diagnostic metrics of your .NET application in Kubernetes to Logz.io, using dotnet-monitor and OpenTelemetry. logzio-dotnet-monitor runs as a sidecar in the same pod as the .NET application.
+
+###### Sending logs from nodes with taints
+
+If you want to ship logs from any of the nodes that have a taint, make sure that the taint key values are listed in your in your daemonset/deployment configuration as follows:
+  
+```yaml
+tolerations:
+- key: 
+  operator: 
+  value: 
+  effect: 
+```
+  
+To determine if a node uses taints as well as to display the taint keys, run:
+  
+```
+kubectl get nodes -o json | jq ".items[]|{name:.metadata.name, taints:.spec.taints}"
+```
+
+#### Standard configuration
+
+<div class="tasklist">
+
+##### Select the namespace
+
+This integration will be deployed in the namespace you set in values.yaml. The default namespace for this integration is logzio-dotnet-monitor.
+
+To select a different namespace, run:
+
+```shell
+kubectl create namespace <<NAMESPACE>>
+```
+
+* Replace <<NAMESPACE>> with the name of your namespace.
+
+
+##### Add `logzio-helm` repo
+  
+```shell
+helm repo add logzio-helm https://logzio.github.io/logzio-helm
+helm repo update
+```
+
+
+###### Run the Helm deployment code
+
+```shell
+helm install -n <<NAMESPACE>> \
+--set secrets.logzioURL='<<LISTENER-HOST>>:8053' \
+--set secrets.logzioToken='<<PROMETHEUS-METRICS-SHIPPING-TOKEN>>' \
+--set-file dotnetAppContainers='<<DOTNET_APP_CONTAINERS_FILE>>' \
+logzio-dotnet-monitor logzio-helm/logzio-dotnet-monitor
+```
+  
+{% include log-shipping/listener-var.html %} {% include log-shipping/log-shipping-token.html %}
+* Replace `<<DOTNET_APP_CONTAINERS_FILE>>` with your .NET application container file. Make sure your main .NET application container has the following volumeMount:
+
+```yaml
+volumeMounts:
+  - mountPath: /tmp
+    name: diagnostics
+```
+
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, then open [Logz.io](https://app.logz.io/). You can filter for data of type `dotnet-monitor-collector` to see the incoming logs.
```

Comment:
Please change "logs" to "metrics" and instead of the last line, please write "You can search for your metrics in Logz.io by searching {job="dotnet-monitor-collector"}"